pubspec.yaml

```diff
@@ -6,7 +6,7 @@ version: 0.3.4
 homepage: https://github.com/dart-lang/io
 
 environment:
-  sdk: ">=2.0.0 <3.0.0"
```

Comment:
Bump the package version to 0.3.5-dev – ideally include the min sdk bump in the changelog

Reply:
Done